src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/subheadline/index.js

```diff
@@ -9,7 +12,7 @@ const convertToSubheadline = async block => {
       blocks: [innerParagraph],
     });
   }
-  console.log(`Incorrect block type ${block.type}`);
+  logger.error(`Incorrect block type ${block.type}`);
```

Comment:
Can this be a warning instead of an error?

Reply:
It would be an error if the subheadline transformer ran on a raw image

Reply:
Of course, makes sense 👍